### PR TITLE
Bento 1289 add age filter to the sidebar section

### PR DESCRIPTION
**Pull request by wfy1997**
## Description

Add age filter to the sidebar section.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/pages/dashboardTab/store/dashboardReducer.js [hunk 1]
```diff
@@ -770,13 +787,74 @@ so it contains more information and easy for front-end to show it correctly.
...
+      datafield: mapping.datafield,
+      show: mapping.show,
+      slider: mapping.slider,
+      quantifier: mapping.slider,
+      section: mapping.section,
+    }))
+  );
+}
+
+/* function setSelectedVlauesToTrue1(checkboxItems, filters) {
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/store/dashboardReducer.js:813]: Can you cleanup the code

Comment (wfy1997) [reply src/pages/dashboardTab/store/dashboardReducer.js:813]: Sorry, let me clean it up

File: src/components/ActiveFiltersQuery/ActiveFiltersQuery.js [hunk 2]
```diff
@@ -13,8 +13,9 @@ const ActiveFiltersQuery = ({ classes }) => {
 
   // activeFilters helps filtering only active Filters
   const activeFilters = [];
-
-  // Loop all filters to find active ones only
+  // Loop all filters except age to find active ones only
+  const ageData = allFiltersinfo.data.filter((sideBar) => sideBar.slider === true);
```
Comment (AjayDoddapaneni) [inline src/components/ActiveFiltersQuery/ActiveFiltersQuery.js:17]: As you are cleaning can you rename the variable name

Comment (wfy1997) [reply src/components/ActiveFiltersQuery/ActiveFiltersQuery.js]: The variable is renamed.

File: src/pages/dashboardTab/store/dashboardReducer.js [hunk 3]
```diff
@@ -899,10 +943,14 @@ const reducers = {
     isDashboardTableLoading: false,
   }),
   TOGGGLE_CHECKBOX_WITH_API: (state, item) => {
-    const updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
+    let updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
```
Comment (bremonline) [inline src/pages/dashboardTab/store/dashboardReducer.js:946]: Codacy has a fix for the issue: [Unexpected newline after '('.](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8502145)
```suggestion
    let updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(item.data.searchSubjects, facetSearchData,
```

**Pull request closed without merging**